### ISSUE TITLE
508-defect-2 [COGNITION]: Heading levels SHOULD increase by one in a logical order

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#17846]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

<hr/>

## Point of Contact

**VFS Point of Contact:** _Josh_

## User Story or Problem Statement

As a screen reader user, I want to navigate the hierarchy of the page content using heading levels to save time and frustration.

## Details

Headings on the page should increase in a logical order. Currently, the use of headings is inconsistent-- most likely due to the form system. Issues include:
- Accordion titles are not `h3`s.
- Sub-section titles within accordions are currently `h2`s which put them at the same level as "Step 4 of 4..." when they should be `h4`s instead.
- Error messages are `h3`s which further complicate the ordering of headings. 

## Acceptance Criteria

- [ ] Headings increase in a logical order

## Environment

* Operating System: any
* Browser: any
* Screenreading device: any
* Server destination: staging

## Steps to Recreate

1. Enter `https://staging.va.gov/decision-reviews/higher-level-review/request-higher-level-review-form-20-0996/review-and-submit` in browser.
2. Complete test case 4 and reach the review page.
3. Inspect the headings on the page and confirm they are not in a logical order.

## Solution 

Several changes need to be made including:
- Accordion titles should be made into `h3`s.
- Sub-section titles e.g. "Contact information" should be made into `h4`s.
- Headings should be added to sub-sections without titles e.g. Issues for Eligible Review (h3) -> Selected Issues (h4), Request an Informal Conference (h3) -> Requested Informal Conference (h4).
- Errors should not be headings.

## WCAG or Vendor Guidance (optional)

* [Headings - Web Accessibility Tutorials](https://www.w3.org/WAI/tutorials/page-structure/headings/)

## Screenshots or Trace Logs

![image](https://user-images.githubusercontent.com/14154792/103014613-0d99b280-450d-11eb-97a4-64e7a4c05632.png)

**Comment by Mottie** [comment]
I can modify the "Contact information" `h2` since that is within HLR's `ReviewDescription` component, but the accordion headers are handled in the platform code.

**Comment by joshkimux** [comment]
@Mottie If we can't modify the accordion headers, we'll probably have to make "Contact Information" a `h3`; thanks for tagging this over to the design system team!

**Comment by Mottie** [comment]
Modifying the accordions to include headers would require a change to the design system - https://design.va.gov/components/accordions

**Comment by joshkimux** [comment]
@caw310 would we be able to get this looked at for the design system?
@Mottie since we can still modify the "Contact Information" in the review description component, could we set that to a `h3` so it logically falls under the `h2` "Review Application" ?

**Comment by Mottie** [comment]
I have a PR already open changing the contact information header :D

And I'm in the process of opening a new design system feature request

**Comment by Mottie** [comment]
Design system ticket - https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/368

**Comment by Mottie** [comment]
The "Contact information" `h3` change has been implemented.

**Comment by joshkimux** [comment]
Confirmed this has been updated! Thank you @Mottie 🥳
